lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -55,6 +55,7 @@ docopt==0.6.2 ; python_version >= "3.7" and python_version < "3.11"
 docutils==0.16 ; python_version >= "3.7" and python_version < "3.11"
 ecdsa==0.18.0 ; python_version >= "3.7" and python_version < "3.11"
 edam-ontology==1.25.2 ; python_version >= "3.7" and python_version < "3.11"
+email-validator==1.3.0 ; python_version >= "3.7" and python_version < "3.11"
```

Comment:
Seems unused? Also in other 2 places below.

Reply:
Turns out it was used by pydantic :)